blockly-core/core/ui/contract_editor/contract_editor.js

```diff
@@ -228,23 +230,24 @@ Blockly.ContractEditor.prototype.setSectionHighlighted = function (viewToHighlig
  * Hides a set of blocks
  * @param isVisible whether to set blocks in area visible (true) or invisible (false)
  * @param blockFilter subset of blocks to look at
- * @param hiddenBlockArray array containing currently hidden blocks, gets filled with
- *                        newly hidden blocks if any are hidden
+ * @param hiddenBlockArray array containing currently hidden blocks
+ * @returns array newly hidden blocks if any are hidden
  */
 Blockly.ContractEditor.prototype.setBlockSubsetVisibility = function(isVisible, blockFilter, hiddenBlockArray) {
+  var newlyHidden = [];
```

Comment:
maybe s/newlyHidden/nowHidden, or something along those lines, since newly implies they were definitely not hidden before (but this array will have both already hidden, and newly hidden)